README.md

```diff
@@ -14,25 +15,24 @@
 </p>
 
 <p align="center">
-   The Design System that powers Razorpay
+   <b>Design System that powers Razorpay⚡  </b>
 <p align="center">
 
 ### [`@razorpay/blade`](https://github.com/razorpay/blade/tree/master/packages/blade)
+![Github Repo Size](https://img.shields.io/github/repo-size/razorpay/blade?style=for-the-badge&color=aqua)
```

Comment:
While this might be great for some repos, in Blade's case, I don't see any advantage that we will get, and it might mislead people. Hence would request if you could remove this change.